### PR TITLE
Circle 2 migration

### PR DESCRIPTION
**Pull request by raphaelvelez**
## Ticket 
ATOM-7777

Migrating all repos to circle ci 2.0 due to 1.0 no longer being supported.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+          SURETAX_POST_PATH: "/Services/V01/SureTax.asmx/PostRequest"
+          SURETAX_CANCEL_PATH: "/Services/V01/SureTax.asmx/CancelPostRequest"
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            - suretax-{{ checksum "Gemfile.lock" }}
+      - run: bundle check --path=vendor/bundle || bundle install --path=vendor/bundle --jobs=4 --retry=3
```
Comment (pedromartinez) [inline .circleci/config.yml]: I thought it was `--jobs 4` instead of --jobs=4`. Do both work?

Comment (raphaelvelez) [reply .circleci/config.yml]: I had a brain fart and typed it out like that and it worked in circle ci.

Comment (pedromartinez) [reply .circleci/config.yml]: 👍

**Comment by raphaelvelez** [comment]
YOU RIGHT

**Comment by starlight849** [comment]
Please take me off this list! I applied for a job a few years ago but do not work here. I will delete this trail.

Thanks!
Jess

**Comment by ufchrisg** [comment]
@starlight849 Unwatch the repo by clicking the `unwatch` at top of page
image

**Pull request merged**